obs-studio-server/source/util-crashmanager.cpp

```diff
@@ -437,16 +437,15 @@ bool util::CrashManager::TryHandleCrash(std::string _format, std::string _crashM
 	// telling the user that he is using too much cpu/ram or process any Dx11 message and output
 	// that to the user
 
-	// If we cannot destroy the obs and exit normally without causing a crash report,
-	// proceed with a crash
-	try {
-		// If for any reason `destroyOBS_API` crashes, the crash recursion is handled
-		OBS_API::destroyOBS_API();
-		exit(0);
-	} catch (...) {
-		util::CrashManager::HandleCrash(_crashMessage);
-	}
-
+	// If we cannot destroy the obs kill the process without causing a crash report,
+	// proceed with it
+	DWORD pid = GetCurrentProcessId();
+	HANDLE hnd = OpenProcess(SYNCHRONIZE | PROCESS_TERMINATE, TRUE, pid);
+	TerminateProcess(hnd, 0);
```

Comment:
I would check for the return value of OpenProcess to make sure the handle is not null.

Reply:
Done!